wsd-frontend/src/components/wsd/LeftColumn/LeftColumn.tsx

```diff
@@ -32,6 +34,9 @@ export async function LeftColumn() {
           </CategoryLink>
         ))}
       </div>
+      <div>
+        <FeelingLuckyButton />
```

Comment:
Maybe we can add the random post to the top? Like 9gag's 
![image](https://github.com/user-attachments/assets/fec4d94d-b27a-496b-a84f-9dd3dc40408e)
navbar, we'd start with a random post link?

Reply:
I think "Feeling Lucky" has a bit more charm than shuffle. Its not like we are a spotify.

But if you think over the placement i can do that @isik-kaplan